html/semantics/embedded-content/the-iframe-element/load-event-timing.window.js

```diff
@@ -0,0 +1,58 @@
+async_test(t => {
+  const frame = document.createElement("iframe");
+  t.add_cleanup(() => frame.remove());
+  let happened = []
+  frame.onload = t.step_func(() => happened.push("load"));
+  document.body.append(frame);
+  happened.push("append");
```

Comment:
Given the possible option of firing an event synchronously during append, how about `happened.push("afterappend")` here and `happened.push("beforeappend")` above to capture that?

Reply:
It definitely needs to fire synchronously for load to end up first in the list (as is expected here). I don't think pushing something before would further clarify that. Seeing when it fires relative to mutation events would be interesting, but I didn't really want to get into that until asked.

Reply:
Yes, you're right, the thing I asked for is already done.